README.md

```diff
@@ -471,12 +472,29 @@ __AWESOME__ apps counter: __135__ 🎉
 - [x] [GitLab](https://github.com/Swordfish90/Lemuroid)
 - [ ] Official page
 
+## Email Clients
+### Fair Mail
+<img alt="Fair Mail" height="64" src="https://raw.githubusercontent.com/M66B/FairEmail/master/app/src/main/res/mipmap-hdpi/ic_launcher.png">
```

Comment:
Delete space between.

Reply:
Nope, replace it by an `E` and lower-case the `M` :wink: (it's "FairEmail", not "FairMail")

Reply:
Wow, you're right. We're both blind haha.
So change `Fair Mail` to `Fair`Email`.